app/views/answer/leader_board.html.erb

```diff
@@ -0,0 +1,37 @@
+
+<div class = "tables">
+
+<h1>Leaderboard</h1>
+<h2>Top 10 scores</h2>
+<p>
+<table class="table table-striped table-bordered table-condensed">
+<thead class="thead-dark">
+<tr>
+  <th class="text-center">Name</th>
+  <th class="text-center">Quiz</th>
+  <th class="text-center">Score</th>
+  <th class="text-center">Date</th>
+</tr>
+</thead>
+<tbody>
+  <% @user_quizzes.each do |user_quiz| %>
+    <tr>
+      <td>
+        <p>
+        <%if UserQuiz.find_by(user_id: user_quiz.user_id).user.image.attached?%>
```

Comment:
Don't add queries in HTML, it should go in controller or models. Please change it.